hpedockerplugin/hpe_storage_api.py

```diff
@@ -221,6 +240,102 @@ def volumedriver_remove(self, name):
             return response
         return json.dumps({u"Err": ''})
 
+    def _get_snapshot_by_name(self, snapshots, snapname):
+        idx = 0
+        for s in snapshots:
+            if s['name'] == snapname:
+                return s, idx
+            idx = idx + 1
+        return None, None
+
+    def volumedriver_remove_snapshot(self, volname, snapname):
+        try:
+            LOG.debug("volumedriver_remove_snapshot - locking volume %s"

```

Comment:
you can make to LOG.info() since for code tracing it will help if the logging is restrictive.